frappe/public/js/frappe/form/multi_select_dialog.js

```diff
@@ -65,25 +71,33 @@ frappe.ui.form.MultiSelectDialog = Class.extend({
 			title: __("Select {0}", [(this.doctype=='[Select]') ? __("value") : __(doctype_plural)]),
 			fields: fields,
 			primary_action_label: __("Get Items"),
+			secondary_action_label: __("Make " + me.doctype),
```

Comment:
```suggestion
			secondary_action_label: __("Make {0}", [me.doctype]),
```
**reason:**
- Translation of only `Make {0}` is much easier & reusable than the translation of`Make SomeDoctypeName`.

Also, I guess doctype name does not require translation here.

Reply:
Done